### DISCUSSION TITLE
Welcome to Centauri-React-Native-Cohort Discussions!

### DISCUSSION THREAD

**Discussion by ankushdharkar** [RealDevSquad/Centauri-React-Native-Cohort#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shreya-mishra** [comment]
👋🏻This is Shreya Mishra , I am a mern stack developer . My preferable tech stack is Rn , RJS . Have almost 1 year of experience in collaborating with some of the people  in making RN applications that is published on playstore as well (Eats app / Business Tracker / Khabri audio platform ) and also worked on some personal projects in React JS. Wanna deep dive into RN - typescript and aim to build one application of my own and publish it on playstore/appstore .  Unfortunately😛currently working in Infosys as an automation developer . Apart from that I will try to give most of my time to RDS community and learn with you guys and will also work on open source project of RDS 😊

**Comment by pallabez** [comment]
Hey everyone👋, I'm Pallab and I'm currently learning React Native and Node js. Fairly new to development and this industry as a whole. My goal was to become a good front-end developer but thanks to @ankushdharkar, I'm currently doing all sorts of development now. 😛 Out of this cohort, I want to have the confidence to build any app from scratch with the proper way of doing things.

**Comment by lyqht** [comment]
Hello everyone, I am Estee. I have last learnt React Native 4 years ago for an internship (where RN is 0.58). I have forgotten many things since then, and keen to learn to be good with newer versions of RN with the community 😄 Let's have fun learning and helping each other!

**Comment by palakgupta2712** [comment]
👋🏽 Hey, I'm Palak Gupta. A full-stack software developer with a passion for design and more inclined towards front-end development. Building projects and learning different kinds of stuff from over last 1 -1.5 years.
 📚 My current stack is - HTML5, CSS3, JavaScript(ES6), React.js, Material-UI styled-components, MongoDB, AWS Amplify, Figma. I'm here to try my hands on React Native and also to learn how to learn and grow in the community. Happy Learning !! ♥🎊

**Comment by jatin33** [comment]
Hey, I'm Jatin Panjwani. Frontend engineer from last 2 years and have worked at multiple startups. My current stack - React, NextJS, Javascript, TailwindCSS. Besides work, I like playing badminton though I kind of suck at it 😆  Here to extend my React capabilites and try my hand in mobile development. Let's learn and grow together 💟 🌳
